src/css/doc.css

```diff
@@ -640,7 +640,7 @@ body {
 
 .doc .olist li,
 .doc .ulist li {
-  margin-bottom: 0.5rem;
+  margin-bottom: 0;
```

Comment:
This will change the appearance of _all_ unordered lists in the docs. If we want to update the lists in the home page, or just the 'Keep exploring' card, we should create a more specific rule that applies to those link - a rule for something like `body.docs-ndl .ulist li...` or `body.docs-ndl div.lists .ulist li...` in _docs-ndl.css_

Reply:
I think the Keep exploring card is barely being used and the places where it still exists need to be updated, but if this affects the content in the doc pages in a disruptive way, then yes. Do you think that is the case?

Reply:
Yes, this would apply to any page generated from the UI bundle because we wrap the page contents in `<article class="doc">...</article>`

Reply:
You think a 0.5rem difference could be disruptive to the rest of the docs? If so, then it's better to create a new rule like you suggested

Reply:
I think it makes list items more readable when the content wraps onto multiple lines. Without it, the spacing between the lines of a list item and the spacing between two list items are pretty much the same. The effect is similar to what would happen if there was no margin below a paragraph.

Reply:
Ok, so I will fix it as you suggested!